deploy/local-path-storage.yaml

```diff
@@ -66,6 +66,14 @@ spec:
         - start
         - --config
         - /etc/config/config.json
+        - --service-account-name
```

Comment:
Those are default values. We don't need to add them explicitly? We can comment them out but leave them here, to let others know the existence of the options.

Reply:
In fact these are required when deploying straight from yaml, not the helm chart.
As you can see in https://github.com/rancher/local-path-provisioner/blob/master/main.go#L174 it errors out if flag is not set. So either handle empty flags with default values or pass in the flags by default,